### PR TITLE
Add Morocco regions

### PR DESCRIPTION
**Pull request by nelmalki**

### CODE REVIEW

**Review by jsanz — COMMENTED** [review]
Hi @nelmalki, thanks for your contribution and for signing the CLA.

Your PR looks really well. I have though a couple of comments:

* Your data definition gathers in the same dataset both provinces and regions of Morocco. That's not ideal since you have then overlapping polygons and we prefer to have different datasets for the different hierarchy administrative levels.   Take a look at how Belgium [regions](https://github.com/elastic/ems-file-service/blob/04978e10bb8877972f7d692a9e29e2da3be109c4/sources/be/regions.hjson#L57-L91) and [provinces](https://github.com/elastic/ems-file-service/blob/04978e10bb8877972f7d692a9e29e2da3be109c4/sources/be/provinces.hjson#L57-L91) are defined. Other examples of this practice are Spain, Italy, South Korea, or the United States of America.
* Also, we accept new datasets because they have the freedom to also get labels in countries official languages. I can imagine for Morocco at least adding the Arabic (`ar`) language along with the English one. Maybe also Moroccan Arabic (`ary`)? 

What do you think?

**Comment by nelmalki** [comment]
> Hi @nelmalki, thanks for your contribution and for signing the CLA.
> 
> Your PR looks really well. I have though a couple of comments:
> 
> * Your data definition gathers in the same dataset both provinces and regions of Morocco. That's not ideal since you have then overlapping polygons and we prefer to have different datasets for the different hierarchy administrative levels.   Take a look at how Belgium [regions](https://github.com/elastic/ems-file-service/blob/04978e10bb8877972f7d692a9e29e2da3be109c4/sources/be/regions.hjson#L57-L91) and [provinces](https://github.com/elastic/ems-file-service/blob/04978e10bb8877972f7d692a9e29e2da3be109c4/sources/be/provinces.hjson#L57-L91) are defined. Other examples of this practice are Spain, Italy, South Korea, or the United States of America.
> * Also, we accept new datasets because they have the freedom to also get labels in countries official languages. I can imagine for Morocco at least adding the Arabic (`ar`) language along with the English one. Maybe also Moroccan Arabic (`ary`)?
> 
> What do you think?

Thanks for your feedkbacks, adaptations in progress !

**Comment by jsanz** [comment]
@nelmalki thanks for the update. I understand you are only interested then in Morocco regions and not provinces.

I'll review this this week.

**Comment by nelmalki** [comment]
> @nelmalki thanks for the update. I understand you are only interested then in Morocco regions and not provinces.
> 
> I'll review this this week.

Thanks @jsanz, I encountred multiple missing data at provinces level (not due to missing P150 property). I pushed this update while trying to stabilize the provinces layer

File: sources/ma/regions_v82.hjson [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+            }
+            }
+            '''
+    }
+    name: morocco_regions
+    legacyIds: [
+      Morocco Regions
+    ]
+    humanReadableName: {
+      ar: جهات المغرب
```
Comment (jsanz) [inline sources/ma/regions_v82.hjson:99]: I see you tweaked these labels, thanks!

**Review by jsanz — CHANGES_REQUESTED** [review]
Just a change so this dataset is also available for <8 clients but other than that, this looks good to me. Please fix that and we are good to go.

File: sources/ma/regions_v82.hjson [hunk 2]
```diff
@@ -0,0 +1,148 @@
+{
+    versions: '>=8.2',
```
[the remark this answers is not in the document]
Comment (nelmalki) [reply sources/ma/regions_v82.hjson]: all good to go !

**Comment by jsanz** [comment]
@kibanamachine test this

**Review by jsanz — APPROVED** [review]
Looks good to me, thanks @nelmalki for your contribution :heart:

**Comment by jsanz** [comment]
@nelmalki after the small fix at #259 I've deployed the last changes and are available for all Kibana users

https://maps.elastic.co/#file/morocco_regions

image


Thanks again for your contribution

**Pull request merged**